joystick.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python3
+
+import os, random
+import robomodules as rm
+from messages import *
+import pygame, sys
+from pygame.locals import *
+
+ADDRESS = os.environ.get("BIND_ADDRESS","localhost")
+PORT = os.environ.get("BIND_PORT", 11297)
+
+FREQUENCY = 20
+
+global logitech
+global x
+global y
+global z
+global pitch
+global yaw
+global roll
```

Comment:
Unless you're writing multi-threaded code, you shouldn't need to use globals. Make these instance variables of the GUIModule class instead.
Eg:
```
def __init__(self, addr, port):
    ...
    self.logitech = None
    self.x = 0
    self.y = 0
    self.z = 0
    ...
```